internal/e2e/metadata_test.go

```diff
@@ -0,0 +1,189 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/cloudoperators/heureka/internal/api/graphql/graph/model"
+	"github.com/cloudoperators/heureka/internal/entity"
+	"github.com/cloudoperators/heureka/internal/server"
+	"github.com/cloudoperators/heureka/internal/util"
+	util2 "github.com/cloudoperators/heureka/pkg/util"
+
+	"github.com/machinebox/graphql"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	testUniqueUserId    = "1"
+	testUserType        = entity.HumanUserType
+	testUserName        = "Joe"
+	testUpdatedUserName = "Donald"
+	testCreatedBy       = "Creator"
+	testUpdatedBy       = "Updater"
+	dbDateLayout        = "2006-01-02 15:04:05 -0700 MST"
+)
+
+var ()
+
+func createUser(port string) {
```

Comment:
It would be nice if we use this function in the `user_query_test.go` too. I would remove the expect checks from the helper function here and only add it to the test that calls this helper function so that its not hidden when a test runs

Reply:
I am not sure we should remove expect checks from helper function/test framework function. Usually common test functions are checking for invalid operation, raising assertions and stopping the test execution.

Reply:
ahh yes. Now I get it. You are right, It is not good idea to hide expects in test for user create, so I would like to keep this function (createUser with expects), but it will use another function which is named queryCreateUser (without expects).

Reply:
Done